server/datastore/mysql/software.go

```diff
@@ -1089,6 +1118,7 @@ func (ds *Datastore) SoftwareByID(ctx context.Context, id uint, includeCVEScores
 			"s.release",
 			"s.vendor",
 			"s.arch",
+			"s.extension_id", // TODO: validate that this is required
```

Comment:
@getvictor same here, I assume `extension_id` was meant to be added (or at least there's no harm in doing so).

Reply:
Yes, thank you.